Weather/Views/TodayWeatherView.swift

```diff
@@ -21,15 +21,15 @@ struct TodayWeatherView: View {
                     .frame(width: 100, height: 100)
                 
                 VStack(alignment: .leading) {
-                    Text("\(cityVM.temperature)°C")
+                    Text("\(cityVM.temperature)" + NSLocalizedString("Temperature unit", comment: ""))
```

Comment:
Please, use localisation with parameters in this case because in some languages the order of number and abreviation can be different. The localizable file should have such key: `temperatureUnit %@` = `%@ °F`. And you should use it like this: `Text("temperatureUnit \(cityVM.temperature)")`